### DISCUSSION TITLE
Allow admin users to create pages

### DISCUSSION THREAD

**Discussion by edulima** [railsadminteam/rails_admin#3530] [Q&A]
One of my clients would like to be able to create pages without a developer intervention to create models, controllers etc. Something similar to what WordPress offers, obviously not as advanced. Is this currently supported? I could not find anything on the docs? Sorry, I haven't use this tool much. I would appreciate if someone could point me to a direction on how to get started on this.

**Comment by edulima** [comment]
@mshibuya are you able to give me some advise on this question? Many thanks

**Reply by codealchemy** [reply]
This library is intended to provide an interface to interact with your underlying models / application; you _could_ put together the appropriate models for your use case and leverage RailsAdmin as the interface to manage the data - but there [may be other gems](https://www.ruby-toolbox.com/categories/Blog_Engines) that could better solve what you're looking to do.
